dashboards-observability/public/components/explorer/docTable/docViewRow.tsx

```diff
@@ -36,6 +35,12 @@ export const DocViewRow = (props: IDocViewRowProps) => {
   const [detailsOpen, setDetailsOpen] = useState<boolean>(false);
   const [surroundingEventsOpen, setSurroundingEventsOpen] = useState<boolean>(false);
   const [openTraces, setOpenTraces] = useState<boolean>(false);
+  const [flyoutToggleSize, setFlyoutToggleSize] = useState(true);
+  const [logTraceId, setLogTraceId] = useState('');
+
+  const isValidTraceId = (traceId: string) => {
```

Comment:
Isn't this function already in the utils.tsx??

Reply:
Thanks will remove the duplicate